src/mod.ts

```diff
@@ -36,6 +36,12 @@ const run = async () => {
 
   options.log(options);
 
+  // Clean output directory
+  if (options.outputDirectory) {
+    await Deno.remove(options.outputDirectory, { recursive: true });
```

Comment:
I tried running this on a fresh project, which didn't have a `dist` folder, and this failed with:

```
11:47:36 in ~/src/hermes-deploy-test via ⬢ v16.14.0
➜ deno run -q --unstable --allow-read --allow-write ../deno-slack-builder/src/mod.ts
{
  manifestOnly: false,
  workingDirectory: "/Users/fmaj/src/hermes-deploy-test",
  outputDirectory: "/Users/fmaj/src/hermes-deploy-test/dist",
  log: [Function: log]
}
error: Uncaught (in promise) NotFound: No such file or directory (os error 2), remove '/Users/fmaj/src/hermes-deploy-test/dist'
    await Deno.remove(options.outputDirectory, { recursive: true });
    ^
    at async Object.remove (deno:runtime/js/30_fs.js:160:5)
    at async run (file:///Users/fmaj/src/deno-slack-builder/src/mod.ts:41:5)

```

Reply:
Fantastic catch @filmaj! I'll make that change today!

Reply:
Commit [0e10673](https://github.com/slackapi/deno-slack-builder/pull/8/commits/0e1067302eaf5e5a0ca1cd0433abff610eff16b0)